README.md

```diff
@@ -30,6 +30,11 @@ To build an individual package in response to changes within the `src` directory
 ```sh
 yarn dev
 ```
+##Documentation
+The following command will generate documentation for each module in packages/sfx using typeDoc. This will update the index file in the docs folder.
```

Comment:
```suggestion
The following command will generate documentation in the `docs` directory for each module in the `packages` directory using [TypeDoc](https://typedoc.org/).
```

Mentioning TypeDoc may not be necessary.

Reply:
```suggestion
The following command will generate documentation for each module in the `packages` directory. It uses [TypeDoc](https://typedoc.org/) and outputs to the `docs` directory.
```

Reply:
(the latter one is a bit less confusing because it's just in two sentences)